cypress/integration/newUserForm.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Login', () => {
+  const app_base_url = Cypress.env('app_base_url')
+  beforeEach(() => {
+    cy.svcClientLogout()
+    cy.svcClientLogin().as('tokens')
+    cy.get('@tokens').then(cy.svcClientSetCookie)
+  })
+
+
+  it('Signs in, and shows privacy message on first login', () => {
+    cy.visit(app_base_url + '/home')
```

Comment:
We can actually set the default base URL for each test by using the `baseUrl` config option in `cypress.json`. Then you would only have to do `cy.visit('/home')`